### PR TITLE
Update Radix Extension

### PR DESCRIPTION
**Pull request by stephan281094**
## Description

- Updated the list of links to the Radix documentation, which includes 3 recently introduced primitives.

Is this change worthy of introducing a `changelog.md` file?

## Screencast

There are no visual changes.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

### CODE REVIEW

File: extensions/radix/src/primitives.tsx [hunk 1]
```diff
@@ -59,19 +59,19 @@ const primitivesRoutes = [
...
       { title: "Radio Group", slug: "docs/primitives/components/radio-group" },
       { title: "Scroll Area", slug: "docs/primitives/components/scroll-area" },
-      { title: "Select", slug: "docs/primitives/components/select", draft: true },
+      { title: "Select", slug: "docs/primitives/components/select", beta: true },
       { title: "Separator", slug: "docs/primitives/components/separator" },
       { title: "Slider", slug: "docs/primitives/components/slider" },
       { title: "Switch", slug: "docs/primitives/components/switch" },
       { title: "Tabs", slug: "docs/primitives/components/tabs" },
-      { title: "Toast", slug: "docs/primitives/components/toast", draft: true },
+      { title: "Toast", slug: "docs/primitives/components/toast", beta: true },
```
Comment (peduarte) [inline extensions/radix/src/primitives.tsx:74]: if you want to rename from `draft` to `beta`, you gotta ensure the list filter is updated too https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L11

**Review by peduarte — CHANGES_REQUESTED** [review]
small comment

**Comment by pernielsentikaer** [comment]
Hi @stephan281094 👋 

I wouldn't make a changelog.md for this, since it's only adding more data 😄  

You could look into [Screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots), it's a new feature that will soon be added, which helps users see what it does 💯 

Thanks for your time in the extension 🔥 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/radix/src/primitives.tsx (hunk 1 above)
Comment (stephan281094) [reply extensions/radix/src/primitives.tsx:74]: Thanks for your comment, Pedro. The original list uses both `draft` _and_ `beta`, so this is not a rename. I figured we probably want to continue hiding drafts, but allow beta primitives to be visible. If you prefer hiding beta primitives as well, I can make that change.

**Comment by stephan281094** [comment]
Thank you, @pernielsentikaer. I did look at the documentation regarding screenshots, but wasn't sure whether it made sense to add. It recommends adding at least 3 screenshots, which seems a bit much for this extension, given that it's just a list of links. Having only one screenshot might make things look a bit awkward in the extension detail screen. Nevertheless, I can add one, let me know!

File: extensions/radix/src/primitives.tsx (hunk 1 above)
Comment (peduarte) [reply extensions/radix/src/primitives.tsx:74]: > The original list uses both draft and beta,

What do you mean by original list?

The primitives types currently in `main` never supported `beta`:
CleanShot 2022-03-29 at 17 14 22@2x

Comment (stephan281094) [reply extensions/radix/src/primitives.tsx:74]: >What do you mean by original list?
I'm referring to https://github.com/radix-ui/website/blob/main/lib/primitivesRoutes.ts. You can see that `beta` is now supported there. But I'm seeing that I didn't update the `PageProps` type accordingly, so I just pushed a fix for that.

**Comment by pernielsentikaer** [comment]
Hi @peduarte 👋 

Do you have any more feedback, or is everything good to go as you see it?

**Comment by pernielsentikaer** [comment]
@stephan281094 I think it would be awesome with one image, if you can? Otherwise, I can help you with the [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) 🔥

File: extensions/radix/src/primitives.tsx (hunk 1 above)
Comment (peduarte) [reply extensions/radix/src/primitives.tsx:74]: what im saying is that you added a new `beta` flag, but you're doing nothing with it.

the `draft` flag was used to filter out items from the list ([as you can see here](https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L11))

the `deprecated` flag is used to show an extra label, ([as you can see here](https://github.com/raycast/extensions/blob/main/extensions/radix/src/primitives.tsx#L16)) like this: 
CleanShot 2022-04-04 at 11 05 17@2x

but the `beta` flag is not being used for anything.

I understand you took it from the radix-site repo, but in there it gets used:
CleanShot 2022-04-04 at 11 06 26@2x


so, if you're gonna add a new flag, I think we need to use it in the UI as a subtitle prop.

Comment (stephan281094) [reply extensions/radix/src/primitives.tsx:74]: Oh I see, I completely missed that, thanks for clarifying! I just updated the pull request to include the "Beta" subtitle. Let me know if there's anything else that needs to be changed.

**Comment by stephan281094** [comment]
@pernielsentikaer Just added a screenshot, great suggestion!

**Comment by pernielsentikaer** [comment]
Hi @stephan281094 👋 

Awesome update, I have a few suggestions for next version 💪 

- Consider adding [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) - it would look something like this:

Ys9yyHY7810SWPCA

- You should maybe invert the logo in dark appearance, it's quite hard to see:

zvZbQXS3v3qLGrQS

You can read more [about it here](https://developers.raycast.com/api-reference/user-interface/icons-and-images#image.source)

**Comment by peduarte** [comment]
@pernielsentikaer since this extension uses a local list of items, it's basically never going to be empty, but TIL about emtpy view! very nice addition to the API.

**Pull request merged**